packages/reactotron-server/src/server/apollo/resolvers/commands.ts

```diff
@@ -0,0 +1,20 @@
+import { Resolver, Query, Subscription, Root } from "type-graphql"
+
+import { MessageTypes } from "../../messaging"
+import { Command } from "../../schema"
+import { commands } from "../../datastore"
+
+@Resolver()
+export class CommandsResolver {
```

Comment:
I am enjoying the splitting of the resolvers. Makes everything much more management IMO